src/Language/Printer.php

```diff
@@ -531,6 +531,6 @@ protected function indent(string $string): string
     /** @param array<string|null> $parts */
     protected function join(array $parts, string $separator = ''): string
     {
-        return \implode($separator, \array_filter($parts));
+        return \implode($separator, \array_filter($parts, static fn (?string $part) => $part !== '' && $part !== null));
```

Comment:
I would rather keep this as is and disable the rule that forbids the simple usage.

Reply:
I'd ignore it locally only for now. In some cases the rule allows no cb

<img width="632" alt="image" src="https://github.com/user-attachments/assets/6de5305e-aace-44b1-99f9-492b3e3db7fd" />